web/public/mgmt/config.yml

```
@@ -2632,6 +2665,31 @@ collections:
               - { label: Hide Button Text Mobile, name: hideButtonTextMobile, widget: string }
               - { label: Locked Pre-TaxId Text, name: lockedPostText, widget: string }
               - { label: Locked Post-TaxId Text, name: lockedPreText, widget: string }
+  - name: "municipal-tasks"
+    label: "Tasks - Municipal"
+    folder: "content/src/roadmaps/municipal-tasks"
+    delete: false
+    create: false
+    summary: "{{filename}} - {{name}}"
+    identifier_field: "{{filename}}"
+    slug: "{{fields.filename}}"
+    fields:
+      - { label: "Internal Notes", name: "notesMd", widget: "markdown", required: false }
+      - { label: "Name", name: "name", widget: "string", required: false }
+      - { label: "ID", name: "id", widget: "write-once-read-only", required: false }
```

[boulder] We do want this to be required
```suggestion
      - { label: "ID", name: "id", widget: "write-once-read-only"}
```